src/cljs/athens/views/blocks/types/default.cljs

```diff
@@ -361,7 +361,7 @@
 
               (when (= @drag-target :before) [drop-area-indicator/drop-area-indicator {:placement "above"}])
 
-              [editor/editor-component
+              [:f> editor/editor-component
```

Comment:
What does this change do?

Reply:
https://github.com/reagent-project/reagent/blob/master/doc/ReactFeatures.md#function-components